src/NServiceBus.Core/Pipeline/WellKnownStep.cs

```diff
@@ -120,5 +119,6 @@ internal static WellKnownStep Create(string customStepId)
         /// Enforces unsubscribe messaging best practices.
         /// </summary>
         public static readonly WellKnownStep EnforceUnsubscribeBestPractices = new WellKnownStep("EnforceUnsubscribeBestPractices");
+        
     }
```

Comment:
:) my bad, first I added the key "LogOutgoingMessages" to WellKnownSteps as I thought that it's a good idea. However then I reverted my implementation by removing line rather than reverting changes.